src/main/java/salesforce/ui/pages/lightning/worktype/WorkTypeDetails.java

```diff
@@ -115,4 +105,51 @@ public String getTxtField(final String nameTxtBox) {
     public String getNameCreatorTxt() {
         return webDriverActions.getTextOfByFieldByLocator(nameCreatorTxt);
     }
+
+    /**
+     * Gets a date when workType is created.
+     *
+     * @return a String to date
+     */
+    @Override
+    public String getCreateDayTxt() {
+        return webDriverActions.getTextOfByFieldByLocator(dateCreateByTxt);
+    }
+
+    /**
+     * Gets a list with values of work type.
+     *
+     * @param table a table with values which decide which text select
+     * @return a list with some data of campaign
+     */
+    @Override
+    public List<String> getValueField(final Map<String, String> table) {
```

Comment:
Since it is a list, the method could be named getFieldValues or getValues or getValuesFromFields.

Reply:
that sound better, i will change for your suggestion